index.html

```diff
@@ -0,0 +1,171 @@
+            Подобрать программу
+          </button>
+        </header>
+      </div>
+    </section>
+
+    <section class="food">
+      <div class="food__wrapper">
+        <h2 class="visually-hidden">
+          Программы питания, которые подойдут Вашему питомцу.
+        </h2>
+        <ul class="food__list">
+          <li class="food__item food__item--slimming">
+            <h3 class="food__title">
+              Похудение
+            </h3>
+            <p class="food__description">
+              Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора на диету! Cat Energy Slim поможет вашему питомцу сбросить лишний вес.
+            </p>
+            <a class="food__catalog" href="#">
+              Каталог slim
+            </a>
+          </li>
+          <li class="food__item food__item--weight">
+            <h3 class="food__title">
+              Набор массы
+            </h3>
+            <p class="food__description">
+              Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy Pro поможет вашему коту нарастить необходимые мышцы!
+            </p>
+            <a class="food__catalog" href="#">
+              Каталог pro
+            </a>
+          </li>
+      </div>
+    </section>
+
+    <section class="work">
+      <div class="work__wrapper">
+        <h2 class="work__title">
+          Как это работает
+        </h2>
+        <ul class="work__list">
+          <li class="work__item work__item--sheet">
+            <p class="work_description">
+              Функциональное питание содержит только полезные питательные вещества.
+            </p>
+          </li>
+          <li class="work__item work__item--powder">
+            <p class="work_description">
+              Выпускается в виде порошка, который нужно лишь залить кипятком и готово.
+            </p>
+          </li>
+          <li class="work__item work__item--reception">
+            <p class="work_description">
+              Замените один-два приема обычной еды на наше функциональное питание.
+            </p>
+          </li>
+          <li class="work__item work__item--time">
+            <p class="work_description">
+              Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!
+            </p>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="example">
+      <div class="example__wrapper">
+        <h2 class="example__title">
+          Живой пример
+        </h2>
+        <p class="example__description">
+          Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день.
+        </p>
+        <div class="example__result">
+          <p class="example__result example__result--title">5 кг</p>
+          <span class="example__result example__result--description">снижение веса</span>
```

Comment:
Почти всегда, когда вкладываешь два блока/элемента с одинаковыми названиями друг в друга - это что плохое.

```suggestion
        <div class="example__result">
          <p class="example__result-title">5 кг</p>
          <span class="example__result-description">снижение веса</span>
```